src/components/base/FilterList/FilterList.tsx

```diff
@@ -0,0 +1,37 @@
+import 'server-only';
```

Comment:
don't forget to:
- remove server-only
- add 'use client'
- use relative imports only

'use client';
```

import React from 'react';
import styles from '@/components/base/FilterList/FilterList.module.scss';
import {
  Filter,
  InterfaceFilterPropTypes,
} from '@/components/base/Filter/Filter';
```